CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Ember Blog
+
+New contributors are welcome! Ember Blog is maintained by an all-volunteer team, and we are thankful for your help.
+
+You can support us in a few different ways:
+
+- Suggest an article for [The Ember Times](https://blog.emberjs.com/tags/newsletter.html)
+- Do a write-up of an article for The Ember Times
+- Complete an [issue](https://github.com/ember-learn/ember-blog/issues) with the label "good first issue" or "help wanted"
+- Open an issue to provide feedback
+
+If you have questions or need help with writing, please visit the `#dev-ember-learning` channel on the [Ember Community Discord](https://discordapp.com/invite/zT3asNS).
+
+
+## Setup
+
+<details>
+<summary>Installation</summary>
+
+1. Fork and clone this repo.
+
+  ```bash
+  git clone git@github.com:<your GitHub handle>/ember-blog.git
+  ```
+
+1. Change directory.
```

Comment:
I think the code blocks broke the numbering scheme.

Reply:
Ah. GitHub knows to auto-increment the numbers. Using `1` always lets us skip manually changing the numbers should the list need to change.

Reply:
It looks like I had to use 4 spaces instead of 2 for the preceding code blocks.